warehouse/templates/index.html

```diff
@@ -64,8 +64,8 @@ <h1 class="title" {{ l20n("homepageIntroduction") }}>Find, install and publish P
     </div>
     <div class="about-text" {{ l20n("homepageAbout") }}>
       <p class="lede-paragraph">The Python Package Index (PyPI) is a repository of software for the Python programming language.</p>
-      <p>PyPI helps you find and install third-party software developed and shared by other members of the Python community. <a href="TODO">Learn about installing packages</a>.</p>
-      <p>Package authors use PyPI to distribute their software.  <a href="TODO">Learn how to package your Python code for PyPI</a>.</p>
+      <p>PyPI helps you find and install third-party software developed and shared by other members of the Python community. <a href="TODO">Learn about installing projects</a>.</p>
```

Comment:
I think this works better as "package" since it's talking about the end user "thing you're going to install".